backend/src/services/external-migration/external-migration-fns.ts

```diff
@@ -141,7 +141,7 @@ export const importDataIntoInfisicalFn = async ({
 
   // Import environments
   if (data.environments) {
```

Comment:
The one major issue i see with this whole setup is consider it failed. It keeps the stale things.

So they can't do it again in same project right? Because consider createEnvironment, previous environment won't allow it. They would have to create a new project for this again.

Now same could get worse for secrets as well.  One secret failing could be quite troublesome